src/simulators/tensor_network/MPS.cpp

```diff
@@ -0,0 +1,375 @@
+/**
```

Comment:
Can we rename this file to lower case? and write some documentation in the header explaining what it's purpose.

Reply:
@atilag ,thanks for excellent comments! I will address them all.

Reply:
Maybe better to name it matrix_product_state.hpp/cpp? (rather than mps.hpp/cpp).
Also for SVD.hpp/cpp, do you think I should change it to svd.hpp/cpp?

Reply:
`matrix_product_state.hpp` and `svd.hpp` are fine.  We are still very much a header only library (but this is something that we may change in the near future), so designing your data structures to be used as .hpp is ok for now.

Reply:
Updated file names to lowercase.

Reply:
Should I add the IBM copyright notice at the beginning of each file?

Reply:
I added some documentation about the general idea and algorithm. I also made some fixes in the indentation and in the style of the comments to be more compatible with the rest of Aer.